src/go/k8s/kuttl-test.yaml

```diff
@@ -23,6 +23,6 @@ commands:
     background: true
   - command: "./hack/wait-for-webhook-ready.sh"
 artifactsDir: tests/_e2e_artifacts
-timeout: 600
+timeout: 900
```

Comment:
I'd rather we make tests fit within 10 minutes. CI already takes way too long.

Reply:
Ok. I will roll back.

Reply:
Done